### PR TITLE
[NB-3062] Update notebooks dropins

### PR DESCRIPTION
**Pull request by cdevent**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.
Notebook kernels has been outdated and may cause a problem to build

### CODE REVIEW

**Comment by yakov-g** [comment]
jenkins run test_integration_general test_integration_python_keras

**Comment by yakov-g** [comment]
jenkins run test_integration_general

**Comment by rvorobii** [comment]
@yakov-g please review this PR

**Comment by rvorobii** [comment]
I assigned @yakov-g to review this PR -- please wait for his review

**Comment by cdevent** [comment]
@yakov-g  just a friendly reminder, if you have any time to review this PR

**Comment by yakov-g** [comment]
jenkins run test_integration_general

File: public_dropin_notebook_environments/python39_notebook/env_info.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-  "description": "This template environment can be used to create Python 3.9 notebook environments.",
-  "programmingLanguage": "python",
-  "environmentVersionId": "6386dc2159c606b0d8beddc8",
-  "isPublic": true,
-  "useCases": ["notebook"]
+	"id": "6386dc1159c606b0d8beddc7",
+	"name": "[DataRobot] Python 3.9 Notebook Drop-In",
+	"description": "This template environment can be used to create Python 3.9 notebook environments.",
+	"programmingLanguage": "python",
+	"environmentVersionId": "6386dc2159c606b0d8beddc4",
```
Comment (yakov-g) [inline public_dropin_notebook_environments/python39_notebook/env_info.json:6]: Looks like you just manually bump or change the last character.
Probably generating a new one is safer. 
Just make sure there is no  clash

**Review by yakov-g — APPROVED** [review]
Just one note regarding `environmentVersionId`

**Comment by yakov-g** [comment]
jenkins run test_integration_general

**Comment by yakov-g** [comment]
jenkins run test_integration_general

**Review by lyapun — APPROVED** [review]

**Pull request merged**